bg_remove_iterative.py

```
@@ -0,0 +1,419 @@
+                                   .replace('{thresh}', str(thresh))
+    r = exec_gimp(scan_code)
+    out_lines = (r.get('results') or [''])[0]
+
+    # Parse BGPTS from output
+    bg_pts = []
+    for line in out_lines.strip().splitlines():
+        if line.startswith('BGPTS:'):
+            try:
+                bg_pts = json.loads(line[6:])
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)